import.sh

```diff
@@ -6,9 +6,11 @@ set -o pipefail
 
 VIA_CURL=""
 IMPORTER="$PWD/importer.py"
-BASE_URL="https://raw.githubusercontent.com/appsembler/msft-courses/master"
+BASE_URL="https://raw.githubusercontent.com/bryanlandia/msft-courses/bryan/feature/set-course-start-end-date"
```

Comment:
@bryanlandia instead of patching the branch and risking forgetting it, here's a slightly better trick:

```bash
$ curl https://raw.githubusercontent.com/appsembler/msft-courses/master/import.sh | sed -e 's|/master|/bryan/feature/set-course-start-end-date|' | bash
```